tests/existing-resources/main.tf

```diff
@@ -0,0 +1,26 @@
+########################################################################################################################
+# Resource group
+########################################################################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.6"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+########################################################################################################################
+# Watsonx Assistant
+########################################################################################################################
+
+module "watsonx_assistant" {
+  source                              = "../../"
+  region                              = var.region
+  watsonx_assistant_name              = "${var.prefix}-wx-assistant"
+  watsonx_assistant_plan              = "plus"
```

Comment:
do we need to use plus plan here? Is there a cheaper plan, or do they have quota limits that we might hit?

Reply:
I included `plus` to provide endpoint information which is not present for `trial` and `lite` plans. 
The `Lite` plan is capped for free exploration purposes​. I think should be either `Trial` or `Lite` for testing purpose. I should change this to `Lite` aka `free`

Here is a small quota info for all plans: 

### Lite Plan (Free):
```
Monthly Active Users (MAU): Up to 1,000.
Assistants: 5.
Published Versions per Assistant: 1.
Session Timeout: 5 minutes.
Analytics Data Retention: 7 days.
```

### Plus Plan ($140/month):
```
Monthly Active Users (MAU): 1,000+ (additional charges apply for higher usage).
Assistants: 10.
Published Versions per Assistant: 10.
Session Timeout: 24 hours.
Analytics Data Retention: 30 days.
```

### Enterprise Plan (Custom Pricing):
```
Monthly Active Users (MAU): 50,000+ (can be increased with a custom agreement).
Assistants: 30.
Published Versions per Assistant: 50.
Session Timeout: Up to 7 days.
Analytics Data Retention: Up to 90 days.
```